core/src/Alert.html

```diff
@@ -0,0 +1,113 @@
+<div
+  class="fd-alert fd-alert--{alertTypeMap[settings.type]} fd-alert--dismissible"
+  role="alert"
+  id="j2ALl423"
+  data-cy="luigi-alert"
+>
+  <button
+    class="fd-alert__close close-button"
+    on:click="fire('alertDismiss')"
+    aria-label="Close"
+    aria-controls="j2ALl423"
+    data-cy="luigi-alert-dismiss"
+  ></button>
+  {@html settings.text}
+</div>
+
+<script type="text/javascript">
+  import * as Routing from './services/routing.js';
+  import * as RoutingHelpers from './utilities/helpers/routing-helpers';
+
+  export default {
+    data() {
+      return {
+        alertTypeMap: {
+          info: 'information',
+          success: 'success',
+          warning: 'warning',
+          error: 'error'
+        }
+      };
+    },
+    oncreate() {
+      const defaultSettings = {
+        text:
+          'I am a Luigi alert with extremely relevant information for the user',
```

Comment:
Great point. I will display an info message in the browser console if an alert has no text and will not display anything in the UI. Additionally, I will remove the default values for alerts, since this makes no sense:
```
      const defaultSettings = {
        type: 'info',
        links: undefined
      };
```